chore: add developer marketer to the careers page (#304)
Redocly/website: pages/careers/developer-marketer.md

---
template: ../../@theme/templates/OpenPosition
title: Developer Marketer (Video-Focused)
locations:
  - Lviv, Ukraine
employmentType: Full time
slug: /careers/developer-marketer
---

#### About the role

> We're looking for a creative and curious video content creator with a marketing focus. Youʼll produce short product and feature videos, demos, and how-tos that help developers discover, understand, and get the most out of our tools. 

Reviewer: ```suggestion
We're looking for a creative and curious technical content creator with a marketing mindset. You’ll produce short videos, product demos, feature walkthroughs, and how-tos that help teams discover, understand, and get the most out of our products and platform features.
```

> #### Key Responsibilities

Reviewer: ```suggestion
#### Key responsibilities
```

- Write short scripts, outlines, and hooks tailored for developer audiences.
- Maintain a content calendar and manage a steady publishing cadence.
- Experiment with new formats, storytelling approaches, and distribution strategies.

#### What we’re looking for
- 2+ years of experience creating product or technical explainer videos.
- Strong skills in video recording, narration, and editing.
- Proficiency with professional video editing tools (ScreenFlow, Final Cut, Descript, CapCut or similar).
- Ability to write short scripts and content outlines.
- Comfortable tracking content metrics and drawing actionable conclusions.
- Have strong written and spoken English.
- Curiosity about technical products and willingness to deeply understand how features work.
- Creativity, not afraid to make a mistake, ask questions, try and test hypotheses.

#### Nice to have
- Experience working with OpenAPI.
- Experience with documentation tooling.
- Familiarity with Redocly products (Reunite, Redoc, Revel, Reef, Realm).

### 30-60-90 day plan

#### First 30 days
- Learn our products and audience.
- Create your first video pair (demo + how-to).

> - Define your workflow for production and publishing.

Reviewer: ```suggestion
- Define your workflow for publishing.
```

> #### Next 30 days

Reviewer: I would also add
```
- Start collaborating with teams to identify upcoming features and use cases worth covering.
```

Reviewer: and 
```
Help define tone, style, and best practices for technical video content.
```

Author: Sound good, I've just added it

- Produce 4 more video pairs.
- Add variation (subtitles, b-roll, improved structure).

> - Experiment with clips and shorts for social media.

Reviewer: ```suggestion
- Experiment with different content formats and hooks.
```

#### Next 30 days
- Continue to produce at least one video pair per week.
- Start measuring impact (views, completion rates, trials).
- Build a simple content calendar and feedback loop.

> - Plan the next wave of features to highlight.

Reviewer: ```suggestion
- Identify top-performing formats and topics.
- Propose the next wave of educational, feature, and launch content.
```
